terraform/modules/services/airflow/docker-compose-0.0.3.yml

```diff
@@ -1,124 +0,0 @@
-version: "3"
```

Comment:
I think we might need this to run the service

Reply:
It doesn't use the 0.0.3.yml in the deployment, just the docker-compose.yml, see https://github.com/openclimatefix/ocf-infrastructure/blob/b6f65cccb4c6ab40da80e6283438679c4e557d9a/terraform/modules/services/airflow/s3.tf#L14